### PR TITLE
Add EC limitation to prometheus topic

### PR DESCRIPTION
**Pull request by paigecalvert**
Adding Custom Graphs: https://deploy-preview-2550--replicated-docs-upgrade.netlify.app/vendor/admin-console-prometheus-monitoring

Monitoring with Prometheus: https://deploy-preview-2550--replicated-docs-upgrade.netlify.app/enterprise/monitoring-applications

### CODE REVIEW

File: docs/enterprise/monitoring-external-prometheus.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Consuming Prometheus Metrics Externally
 
-This topic describes how to consume Prometheus metrics in embedded clusters created by Replicated kURL from a monitoring service that is outside the cluster.
+This topic describes how to consume Prometheus metrics in Replicated kURL clusters from a monitoring service that is outside the cluster.
```
Comment (paigecalvert) [inline docs/enterprise/monitoring-external-prometheus.md:3]: just removing "embedded" throughout this topic

File: docs/enterprise/monitoring-applications.md [hunk 2]
```diff
@@ -1,36 +1,32 @@
-# Monitoring Applications
+# Monitoring Applications with Prometheus
 
 This topic describes monitoring applications and clusters with Prometheus. It includes information about how to configure Prometheus monitoring for existing clusters and how to access the dashboard using a port forward.
 
-## About Prometheus
+## Overview
```
Comment (paigecalvert) [inline docs/enterprise/monitoring-applications.md:5]: just some reorg to make this overview easier to follow

File: docs/partials/monitoring/_limitation-ec.mdx [hunk 3]
```diff
@@ -0,0 +1 @@
+Monitoring applications with Prometheus is not supported for installations with [Replicated Embedded Cluster](/vendor/embedded-overview).
```
Comment (paigecalvert) [inline docs/partials/monitoring/_limitation-ec.mdx:1]: Added to topic about configuring monitoring and about how vendors can add custom graphs

File: docs/vendor/admin-console-prometheus-monitoring.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
+import OverviewProm from "../partials/monitoring/_overview-prom.mdx"
+import LimitationEc from "../partials/monitoring/_limitation-ec.mdx"
+
+# Adding Custom Graphs
```
Comment (paigecalvert) [inline docs/vendor/admin-console-prometheus-monitoring.mdx:4]: only changes were adding the limitation section and replacing the overview with the partial

might have also removed some "embedded" terminology for references to kurl

File: docs/enterprise/monitoring-applications.mdx [hunk 5]
```diff
@@ -1,36 +1,21 @@
...
-This topic describes monitoring applications and clusters with Prometheus. It includes information about how to configure Prometheus monitoring for existing clusters and how to access the dashboard using a port forward.
-
-## About Prometheus
-
-The Replicated KOTS Admin Console uses the open source systems monitoring tool Prometheus to collect metrics on an application and the cluster where the application is installed.
-
-Prometheus uses a multi-dimensional data model with time series data and a flexible query language. Prometheus components include the main Prometheus server, which scrapes and stores time series data, and an Alertmanager for alerting on metrics.
-
-For more information about Prometheus, see [What is Prometheus?](https://prometheus.io/docs/introduction/overview/) in the Prometheus documentation.
+# Monitoring Applications with Prometheus
```
Comment (paigecalvert) [inline docs/enterprise/monitoring-applications.mdx:4]: Add limitation and replaced the overview with a partial that's shared with the matching vendor topic

File: docs/partials/monitoring/_overview-prom.mdx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+For installations in existing clusters and in Replicated kURL clusters, the Replicated KOTS Admin Console uses the open source systems monitoring tool Prometheus to collect metrics on an application and the cluster where the application is installed. Prometheus components include the main Prometheus server, which scrapes and stores time series data, and an Alertmanager for alerting on metrics. For more information about Prometheus, see [What is Prometheus?](https://prometheus.io/docs/introduction/overview/) in the Prometheus documentation.
```
Comment (ajp-io) [inline docs/partials/monitoring/_overview-prom.mdx:1]: There might also be Grafana, but you'd have to verify with the engineers.

File: docs/partials/monitoring/_overview-prom.mdx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+In addition to these default graphs, application developers can also expose business and application level metrics and alerts on the dashboard.
+
+The following screenshot shows an example of the **Monitoring** section on the Admin Console dashboard with the Disk Usage, CPU Usage, and Memory Usage default graphs.
+
+<img alt="Graphs on the Admin Console dashboard" src="/images/kotsadm-dashboard-graph.png" width="600px"/> 
+
+[View a larger version of this image](/images/kotsadm-dashboard-graph.png)
+
+For kURL installations, the Prometheus monitoring system is included by default alongside the installed application. No additional configuration is required to collect metrics and view any default and custom graphs on the Admin Console dashboard.
```
Comment (ajp-io) [inline docs/partials/monitoring/_overview-prom.mdx:17]: If the vendor chooses the Prometheus add-on

**Review by ajp-io — COMMENTED** [review]
There's also a reference to embedded clusters with kURL [here](https://deploy-preview-2550--replicated-docs-upgrade.netlify.app/enterprise/monitoring-applications#access-the-dashboards-with-kubectl-port-forward), which might be confusing.

File: docs/partials/monitoring/_overview-prom.mdx (hunk 6 above)
Comment (paigecalvert) [reply docs/partials/monitoring/_overview-prom.mdx:1]: Looks like it:

>In addition to the Prometheus Expression Browser, Grafana and some preconfigured dashboards are included with Kube-Prometheus for advanced visualization.

https://docs.replicated.com/enterprise/monitoring-applications#about-visualizing-metrics-with-grafana

Comment (paigecalvert) [reply docs/partials/monitoring/_overview-prom.mdx]: Chris confirmed: https://replicated.slack.com/archives/C036BTS7JCE/p1723218427525729?thread_ts=1723218309.349859&cid=C036BTS7JCE

**Pull request merged**